src/site/content/en/index.md

```diff
@@ -2,51 +2,275 @@
-promoPanel:
-  title: 'Chrome Dev Summit 2021'
-  summary: 'Join the Chrome team for workshops, office hours, and learning lounges taking place throughout November.'
-  primaryButtonUrl: 'https://events.withgoogle.com/chrome-dev-summit-2021/registrations/new/details/'
-  primaryButtonText: 'Request an invite'
-  secondaryButtonUrl: 'https://developer.chrome.com/devsummit/schedule/'
-  secondaryButtonText: 'View schedule'
-  image: 'https://web-dev.imgix.net/image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/KwS4L954TzN3KK3pWmtA.svg'
-  imageWidth: '530'
-  imageHeight: '483'
+  alt: 'Celebrate Chrometober with web tricks and treats'
+  theme: 'light'
+
 pickedCaseStudyUrl: '/mishipay/'
-pickedCaseStudyTheme: 'light'
-courses:
-  title: 'Featured courses'
-  summary: 'Explore our structured learning paths to discover everything you need to know about building for the modern web.'
-caseStudies:
-  title: 'Case studies'
-  summary: 'Learn why and how other developers have used the web to create amazing web experiences for their users.'
-  buttonText: 'See all'
-  buttonUrl: '/tags/case-study/'
-podcasts:
-  title: 'Podcasts and Shows'
-  summary: 'Level up your web development skills by listening to podcasts from Google Developers'
-  buttonText: 'See all'
-  buttonUrl: '/podcasts/'
-patterns:
-  title: 'Patterns'
-  summary: 'A collection of code snippets to help you optimize your web projects.'
-  buttonText: 'See all'
-  buttonUrl: '/patterns/'
+
+news:
+  title: Web platform news
+  subTitle: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+learns:
+  - category: CSS and UI
+    cards: 
+      - title: Thinking on ways to solve cafe wall illusion
+        description: Praesent accumsan eros orci quis congue metus porta a sed dapibus magna nunc non finibus erat ullamcorper sit amet.
+        thumbnail: image/SZHNhsfjU9RbCestTGZU6N7JEWs1/MqeKXDtUPO7zszzpRXcF.png
+        eyebrow:
+          icon: featured
+          text: Featured
+        cssClass: 'card-vertical'
+      - title: 'The CSS Podcast: Media query range syntax'
+        description: Praesent accumsan eros orci quis congue metus porta a sed dapibus magna.
+        thumbnail: image/SZHNhsfjU9RbCestTGZU6N7JEWs1/VwL892KEz6bakZMlq10D.png
+        eyebrow:
+          icon: podcast
+          text: Podcast
+        cssClass: 'card-horizontal col-2 bg-tertiary'
+      - title: Meet the Chrome team
+        description: We're meeting you where you are. Join us at upcoming web conferences in your region or catch up on past events.
+        eyebrow:
+          icon: event
+          text: Event
+      - title: How Terra improved user engagement thanks to Dark Mode
+        description: By displaying a dark theme to users that prefer dark mode on their devices, Terra reduced the bounce rate by 60%.
+        thumbnail: image/SZHNhsfjU9RbCestTGZU6N7JEWs1/DUMO47sKvnXAt3QDS2o3.png
+        eyebrow:
+          icon: blog
+          text: Blog
+        cssClass: 'card-vertical row-2 col-3'
```

Comment:
Please come up with a card type instead of asking a uthors to learn individual class names. Please add documentation on web.dev/handbook on how to use cards on homepage (similar to what you put in Pr description).